preprocessing.py

```diff
@@ -138,7 +138,7 @@ def read_tf_records(batch_size, tf_records, num_repeats=None,
     if num_repeats is not None:
         dataset = dataset.repeat(num_repeats)
     else:
-        dataset = dataset.repeat()
+        dataset = dataset.repeat(1)
```

Comment:
just set num_repeats=1 in the default kwargs?

Reply:
XD d'oh, of course.